src/constants.ts

```diff
@@ -59,8 +59,8 @@ export const addresses: any = {
             [Version.V3]: ethers.utils.getAddress(
                 "0x31122CeF9891Ef661C99352266FA0FF0079a0e06"
             ),
-            [Version.V4]: ethers.utils.getAddress(
-                "0xa6EA0E47501627a964e11E31D2BD7D40452dff6F"
+             [Version.V4]: ethers.utils.getAddress(
+                "0xdf85cc2E90b29a42D078B6F24c1AC3B54051E868"
```

Comment:
Update this to match new deployment: https://linear.app/arrow-markets/issue/SC-274#comment-e5a24e28